Here is your resume in GitHub Markdown format:

# Ananya Kaushik

Cambridge, UK  
+44 7436712507 | [ak2470@cantab.ac.uk](mailto:ak2470@cantab.ac.uk)  
GitHub: [github.com/ananyakaushik20](http://github.com/ananyakaushik20)  
LinkedIn: [linkedin.com/in/ananya-kaushik-508b731a3/](http://linkedin.com/in/ananya-kaushik-508b731a3/)

## Bioinformatics Engineer

#### Technical Skills: 
- **Programming Languages**: R, Python, Bash
- **High Throughput Sequencing**: WGS/WXS, bulk RNA-seq, sc-RNA-seq, Microarray, ChIP-seq, ATAC-seq
- **Bioinformatics Analysis**: GWAS, PRS, Computational Drug Design, Protein-Ligand Docking, Gene Co-Expression, Network Analysis, Sequence Assembly & Alignment, Survival Analysis, Mutation and genetic aberration analysis, Motif and domain analysis
- **Clinical Genetics**: Variant Analysis, Annotation, Interpretation, and Classification, Genetic Testing and Genetic Counselling (UKBB, 100K Genomes, DDD, ACMG)
- **Laboratory**: Microscopy, Spectrophotometry, Protein separation techniques, Protein assays, ELISA, Cell Culture, DNA/RNA extraction
- **Tools & Software**: Snakemake, Nextflow, Docker, Singularity, Git, ELNs, APIs, HPC, SQL
- **Statistics and Machine Learning**: Biostatistics, UMAP, PCA, t-SNE, Dimensionality Reduction, Clustering, Linear Regression, RandomForest, DecisionTree, SVM

## Education
- **MPhil in Genomic Medicine** | University of Cambridge (_2022 - 2023_)
  - Masters Research Project: Differential gene expression profiles across Brain Regions and Progression Stages in Alzheimer's Disease
  - _High Pass_

- **B.Sc (Honours) in Biological Sciences** | University of Delhi (_2019 - 2022_)
  - Minor in Statistics
  - Modules: Computational Biology, Genetics, Bioinformatics, etc.
  - _1:1 Grade_

- **PGCert in Bioinformatics and Computational Biology** | University of Delhi (_2020 - 2021_)

## Work Experience

**MPhil Research Placement @ University of Cambridge (_October 2022 - October 2023_)**  
Performed bioinformatics analysis at the Medical Genetic Department, EMBL-EBI, Wellcome Sanger Institute. Responsible for running and troubleshooting NGS analysis pipelines in a high-performance computing environment.

**UG Research Assistant @ CSIR-IGIB (Institute of Genomics and Integrative Biology) (_September 2021 - January 2022_)**  
Curated viral genotype data for an open-access Hep-C viral drug resistance knowledgebase and portal. Analysed population data and genetic variants in viral genomes of different Hep C strains causing drug resistance.

**Research Assistant - Chemogenetics Lab @ Ashoka University (_August 2021 - March 2022_)**  
Conducted a systematic review of emerging gene therapy and drug delivery approaches for Neurodegenerative Diseases like Alzheimer's, Parkinson's, and Huntington's. Shadowed lab staff in molecular biology laboratory techniques.

**Bioinformatics Team - Data Scientist and Pipeline Developer @ STEM-Away, CA (_June 2021 - September 2021_)**  
Conducted Differential Gene Expression Analysis and functional analysis on Lung Cancer and Colorectal Cancer sequencing data. Developed an end-to-end analysis R pipeline and published it as an open-source R shiny web application.

**Research Assistant @ University of Delhi (_June 2020 - September 2021_)**  
Performed in-silico drug discovery and design for Cardiovascular Disease using a pipeline for toxicity prediction, active site prediction, protein-ligand docking, and presented multiple actionable drug targets.

**Summer School Trainee @ CERN (European Centre for Nuclear Research) - CMS (_June 2017 - July 2017_)**  
Participated in a summer school focusing on nuclear research.

## Projects

- **Machine Learning For Drug Discovery**
- **Computer Vision for Medical Imaging**
- **Single Cell Omics Exploratory Analysis**

## Involvement

- **Project Lead @ Mindspar Bioinnovation Challenge, Regional Centre for Biotechnology, New Delhi (_January 2021 - March 2021_)**
- **GWAS and PRS Analysis for Alzheimer’s And Parkinson’s Disease @ Genspace NYC and NIH-National Institute of Aging (_March 2021 - March 2021_)**

## Additional Information

- **GitHub**: [github.com/ananyakaushik20](http://github.com/ananyakaushik20)
- **LinkedIn**: [linkedin.com/in/ananya-kaushik-508b731a3/](http://linkedin.com/in/ananya-kaushik-508b731a3/)

---

This format should provide a well-structured and easy-to-read resume in GitHub Markdown.

## Projects
### Data-Driven EEG Band Discovery with Decision Trees
[Publication](https://www.mdpi.com/1424-8220/22/8/3048)

Developed objective strategy for discovering optimal EEG bands based on signal power spectra using **Python**. This data-driven approach led to better characterization of the underlying power spectrum by identifying bands that outperformed the more commonly used band boundaries by a factor of two. The proposed method provides a fully automated and flexible approach to capturing key signal components and possibly discovering new indices of brain activity.

![EEG Band Discovery](/assets/img/eeg_band_discovery.jpeg)

### Decoding Physical and Cognitive Impacts of Particulate Matter Concentrations at Ultra-Fine Scales
[Publication](https://www.mdpi.com/1424-8220/22/11/4240)

Used **Matlab** to train over 100 machine learning models which estimated particulate matter concentrations based on a suite of over 300 biometric variables. We found biometric variables can be used to accurately estimate particulate matter concentrations at ultra-fine spatial scales with high fidelity (r2 = 0.91) and that smaller particles are better estimated than larger ones. Inferring environmental conditions solely from biometric measurements allows us to disentangle key interactions between the environment and the body.

![Bike Study](/assets/img/bike_study.jpeg)

- [Data Science YouTube](https://www.youtube.com/channel/UCa9gErQ9AE5jT2DZLjXBIdA)
- [Data Science Blog](https://medium.com/@shawhin)
